### ISSUE TITLE
CK sync failing

### ISSUE THREAD

**Issue by mahalakshme** [avniproject/avni-client#1198]
https://avni.freshdesk.com/a/tickets/3257

Looks like the username for whom sync is failing is Dipika. This user sync is incomplete from 17th Nov.

```
select * from sync_telemetry where user_id=1984 order by id desc;
```

**Comment by 1t5j0y** [comment]
Issue due to uuid collision for program_encounter.
Long term fix - #1201 

Short term fix:
- Get user to upload db
- Patch the db locally to update the uuid
- Sync as user using patched db
- Ask user to fresh sync after uninstalling, reinstalling the app.

Patch done on local via Playground.js - [code for this fix.](https://gist.github.com/1t5j0y/a583eb4937c66efb5ee71cc02235bfa7)

Awaiting customer response to implement.

**Comment by 1t5j0y** [comment]
Seeing successful syncs in sync_telemetry for the affected user since Nov 29.
